.opt-in

```diff
@@ -0,0 +1 @@
+pre-commit
```

Comment:
I don't think this file should be committed

Reply:
Oof. Messy commit on my part. Thanks for catching that.